installer/PowerToysSetup/Product.wxs

```diff
@@ -226,7 +226,7 @@
           <Shortcut Id="ApplicationStartMenuShortcut"
                     Name="PowerToys (Preview)"
                     Description="PowerToys - Windows system utilities to maximize productivity"
-                    Directory="ApplicationProgramsFolder"
+                    Directory="ProgramMenuFolder" 
```

Comment:
Might want to also remove the now-unused `ApplicationProgramsFolder`  Directory.

Reply:
upgrade should remove that.  I don't think that is a problem.  Easy enough to test.